### PR TITLE
LRDOCS-4950 through 4952, remaining workflow topics

### PR DESCRIPTION
**Pull request by rbohl**
https://issues.liferay.com/browse/LRDOCS-4951, this addresses managing workflows, uploading workflows, and running an asset through review.

### CODE REVIEW

**Comment by rbohl** [comment]
cc @jhinkey @ngaskill

**Comment by jhinkey** [comment]
Started reviewing

File: discover/portal/articles/140-workflow/03-managing-workflows.markdown [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+
+    You can't unpublish a definition if it's activated for an asset. First
+    dissociate the workflow definition from any assets that use it. See
+    [here](/discover/portal/-/knowledge_base/7-1/activating-workflow) for more
+    information.
+
+## Uploading Workflow Definitions [](id=uploading-workflow-definitions)
+
+If you have an XML definition file locally (perhaps one of the embedded
+workflows), upload it to Liferay Portal:
```
Comment (jhinkey) [inline discover/portal/articles/140-workflow/03-managing-workflows.markdown]: @rbohl I'm not familiar with embedded workflows. Does an example definition really need to be mentioned? Or is there an article we can link to about the embedded workflows?

**Comment by jhinkey** [comment]
Merged into upstream

**Pull request closed without merging**